frida_mode/test/testinstr.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/python3
```

Comment:
`#!/usr/bin/env python3` is (apparently) more portable.
Also I'd suggest running `black` on this file for formatting (should I do it?)